packages/react-docgen-typescript-loader/src/validateOptions.ts

```diff
@@ -17,6 +17,18 @@ const validate = ajv.compile({
       type: "boolean",
     },
 
+    propFilter: {
+      // this is really { type: "function" }
+      not: {
```

Comment:
Not sure about this - propFilter is expected to be a function and ajv has no such thing as `{ type: "function" }` (which is expected since it's built for validation JSON). I've added some checks to simulate checking if it's a function.

Reply:
This seems fine to me. Not much we can do about that.

In the v3 branch, I've swapped `ajv` for `@webpack-contrib/schema-utils` which is still ajv under the hood. Main goal here is to just catch typos during configuration.